uicomponent-compose/feed/src/main/java/io/github/droidkaigi/feeder/feed/FeedScreen.kt

```diff
@@ -280,7 +275,9 @@ private fun AppBar(
                         pagerState.animateScrollToPage(index)
                         tabLazyListStates.getValue(tab).animateScrollToItem(index = 0)
                     }
-                }
+                },
+                // To overwrite the indicator
```

Comment:
Let me change it a little. 🙏 

```suggestion
                // For tabs to draw in front of indicators
```